internal/cmd/operator-sdk/bundle/validate/validate.go

```diff
@@ -111,7 +111,7 @@ func (c bundleValidateCmd) run(logger *log.Entry, bundleRaw string) (res *intern
 			return res, err
```

Comment:
It is missing the changelog fragment. Could you please add?

Reply:
I've added a file and added content to `checklist` section of this PR.